waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -337,3 +337,19 @@
                     :interval 15
                     :timeout timeout-secs)))))
         (log/warn "skipping test as INTEGRATION_TEST_BAD_IMAGE is not specified")))))
+
+(deftest ^:parallel ^:integration-fast test-kubernetes-reverse-proxy-sidecar
+  (testing-using-waiter-url
+    (when (using-k8s? waiter-url)
+      (let [{:keys [service-id] :as response}
+            (make-request-with-debug-info
+              {:x-waiter-distribution-scheme "simple"
+               :x-waiter-name (rand-name)
+               :x-waiter-env-GRPC_TRANSCODER "yes"}
+              #(make-kitchen-request waiter-url % :method :get :path "/"))
```

Comment:
You should be hitting the `/request-info` endpoint to get the request headers coming from envoy.